### PR TITLE
Continuing the edit experience for the user Sprint6/issue#91

### PR DESCRIPTION
**Pull request by nathan96g**

### CODE REVIEW

File: app/src/main/java/ch/epfl/qedit/view/edit/EditFieldFragment.java [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+    }
+
+    /**
+     * static factory method that create a new instance of an EditFieldFragment
+     *
+     * @param isText boolean argument passed through a bundle to the new Fragment
+     * @return a EditFieldFragment with the default parameters set in function of isText
+     */
+    public static EditFieldFragment newInstance(boolean isText) {
+        EditFieldFragment dialogue = new EditFieldFragment();
```
Comment (brunnerant) [inline app/src/main/java/ch/epfl/qedit/view/edit/EditFieldFragment.java:67]: Small typo here: it should be `dialog` ;)

File: app/src/main/java/ch/epfl/qedit/view/edit/EditFieldFragment.java [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+        preview.setEnabled(isPreFilled);
+        preview.setText(""); // empty the previous to see the hint if is not pre filled
+
+        // update checkbox visibility
+        int checkBoxVisibility = isNumber() ? View.VISIBLE : View.GONE;
+        decimalCheckbox.setVisibility(checkBoxVisibility);
+        signCheckbox.setVisibility(checkBoxVisibility);
+    }
+
+    /** @return true if the field want a number answer, False otherwise */
```
Comment (Anthony-Iozzia) [inline app/src/main/java/ch/epfl/qedit/view/edit/EditFieldFragment.java:188]: Typo: "want" -> "wants"

File: app/src/main/java/ch/epfl/qedit/view/edit/EditOverviewFragment.java [hunk 3]
```diff
@@ -41,46 +50,83 @@ public View onCreateView(
...
 
-    private void createAdapter() {
-        addDummyQuestions();
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        if (requestCode == EDIT_QUESTION_ACTIVITY_REQUEST_CODE) {
+            if (resultCode == RESULT_OK) {
+                // Get the question and the extended StringPool from the return data
```
Comment (Anthony-Iozzia) [inline app/src/main/java/ch/epfl/qedit/view/edit/EditOverviewFragment.java:68]: "returned data"

File: app/src/main/res/values-fr/strings.xml [hunk 4]
```diff
@@ -39,6 +39,13 @@
     <!-- Quiz -->
     <string name="title_activity_edit_quiz">Édition</string>
 
+    <!-- Edit New Quiz Settings -->
+    <string name="edit_quiz_title">Entrez le titre du quiz</string>
+    <string name="button_start_editing_message">Démarrer l\'édition</string>
+    <string name="no_question_title_message">Pas de titre</string>
+    <string name="no_question_text_message">Pas de text</string>
```
Comment (Anthony-Iozzia) [inline app/src/main/res/values-fr/strings.xml:46]: "texte"

File: app/src/main/res/values-fr/strings.xml [hunk 5]
```diff
@@ -39,6 +39,13 @@
     <!-- Quiz -->
     <string name="title_activity_edit_quiz">Édition</string>
 
+    <!-- Edit New Quiz Settings -->
+    <string name="edit_quiz_title">Entrez le titre du quiz</string>
```
Comment (Anthony-Iozzia) [inline app/src/main/res/values-fr/strings.xml:43]: "Entrer"

File: app/src/main/res/values-fr/strings.xml [hunk 6]
```diff
@@ -52,12 +59,37 @@
...
+    <!-- Edit Answer -->
+    <string name="choose_answer_format">Choisir un type de réponse tel que TEXTE, NOMBRE, MATRICE et ANNOTATION DE GRAPHE.</string>
+
+    <!-- Edit Field -->
+    <string name="edit_field_title">Edition de champ</string>
+    <string name="preview_title">Aperçu</string>
+    <string name="hint_pre_filled_field">Entrer le texte de ce champ pré-rempli.</string>
+    <string name="hint_text_field">Entrer une réponse textuelle ici.</string>
+    <string name="check_box_sign">Peut être négatif</string>
+    <string name="check_box_decimal">Peut être decimal</string>
```
Comment (Anthony-Iozzia) [inline app/src/main/res/values-fr/strings.xml:79]: "décimal"

File: build.gradle [hunk 7]
```diff
@@ -6,7 +6,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.2'
+        classpath 'com.android.tools.build:gradle:3.6.3'
```
Comment (Anthony-Iozzia) [inline build.gradle:9]: Hmmmmm no. I don't like it.

**Pull request merged**